django/core/validators.py

```diff
@@ -376,6 +377,15 @@ def compare(self, a, b):
         return a < b
 
 
+@deconstructible
+class StepValueValidator(BaseValidator):
+    message = _('Ensure number is of correct step size.')
```

Comment:
This of course is debatable, but how about this message:
`'Ensure this value is a multiple of step size %(limit_value)s.'`